.github/workflows/unit-tests.yml

```diff
@@ -91,6 +91,17 @@ jobs:
         run: |
           bash ./tests/run_code_style.sh mypy
 
+      # Download MNIST: https://github.com/pytorch/ignite/issues/1737
+      - name: Download MNIST
+        uses: pytorch-ignite/download-mnist-github-action@master
+        with:
+          target_dir: /tmp
+
+      - name: Copy MNIST
+        shell: bash -l {0}
+        run: |
+          cp -R /tmp/MNIST .
```

Comment:
why do we need to copy it ?

Reply:
We copy it for examples, do you think refactor examples would be better?

Reply:
Please, put a comment explaining that